src/main/java/mekanism/client/gui/GuiLaserAmplifier.java

```diff
@@ -134,11 +134,15 @@ public void keyTyped(char c, int i) throws IOException {
             }
         }
 
-        if (Character.isDigit(c) || c == '.' || c == 'E' || isTextboxKey(c, i)) {
+        if (Character.isDigit(c) || isTextboxKey(c, i)) {
             minField.textboxKeyTyped(c, i);
             maxField.textboxKeyTyped(c, i);
             timerField.textboxKeyTyped(c, i);
         }
+        else if(c == '.' || c == 'E') {
```

Comment:
why not just wrap timerField in an `if (c!='.' &&  c !='E')`?

Reply:
because i think it looks better like this but i can change that :)

Reply:
ok i changed that.